Online/Core/Teams/PreventDeadStranding.asm

```diff
@@ -0,0 +1,46 @@
+################################################################################
+# Address: 0x8016be28 # Pause_CheckButtonInputsToPause right after checking if
+# player is in sleep after death
+################################################################################
+
+.include "Common/Common.s"
+.include "Online/Online.s"
+
+.set REG_IS_ASLEEP, 4
+
+backup
+
+# r3 holds boolean for is asleep or not (can or cannot take lives)
+mr REG_IS_ASLEEP, r3
```

Comment:
This is not needed. Just make sure you don't use r3 anywhere and it'll just maintain the same value unless you overwrite it.

Reply:
oops, that's true.